### PR TITLE
[Lock] Describe reliability in Lock

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: components/lock.rst [hunk 1]
```diff
@@ -253,6 +253,227 @@ Instead of the simple majority strategy (``ConsensusStrategy``) an
...
+
+If the Redis service or the machine hosting it restarts, every locks would
+be lost without notify running processes.
+
+.. caution::
+
+    To avoid that someone else acquires a lock after a restart, we recommend
+    to delayed service start and wait at least as long as the longest lock TTL.
+
+.. tips::
```
Comment (xabbuh) [inline components/lock.rst]: `.. tip::`

File: components/lock.rst [hunk 2]
```diff
@@ -253,6 +253,227 @@ Instead of the simple majority strategy (``ConsensusStrategy``) an
...
+The ``CombinedStore`` will be, at best, as reliable than the less reliable of
+all managed stores. As soon as one managed store returns erroneous information,
+the ``CombinedStore`` would be not reliable.
+
+.. caution::
+
+    All concurrent processes MUST use the same configuration. with the same
+    amount of managed stored and the same endpoint.
+
+.. tips::
```
Comment (xabbuh) [inline components/lock.rst]: same here

**Comment by javiereguiluz** [comment]
Thanks Jérémy.

File: components/lock.rst (hunk 1 above)
Comment (jderusse) [reply components/lock.rst]: fixed. Sorry for the delay

**Pull request closed without merging**